src/throttler.guard.ts

```diff
@@ -141,8 +141,8 @@ export class ThrottlerGuard implements CanActivate {
     ttl: number,
   ): Promise<boolean> {
     const { req, res } = context.getArgByIndex(2);
-    // add in escape route for GQL Fastify
-    // and for if res is not added to the context.
+    // Add in escape route for GQL Fastify.
+    // And if res is not added to the context.
```

Comment:
This is just 1 sentence,  right? Why not just do `Return early for GQL Fastify or if the res doesn't exist.` ? Sentences never start with `And...` so this comment looks exceptionally weird now.

Reply:
Got it. Resolved. Gonna go ahead and merge based on discussion from Discord.